compute/resource_instance_pool.go

```diff
@@ -61,6 +61,8 @@ func ResourceInstancePool() *schema.Resource {
 		s["custom_tags"].ForceNew = true
 		s["enable_elastic_disk"].ForceNew = true
 		s["enable_elastic_disk"].Default = true
+		s["aws_attributes"].ConflictsWith = []string{"azure_attributes"}
+		s["azure_attributes"].ConflictsWith = []string{"aws_attributes"}
```

Comment:
no gcp attributes for instance pools?

Reply:
there is no instance pools support yet, so I can't test it - will add later